### ISSUE TITLE
Remove shipments after rejection automatically.

### ISSUE THREAD

**Issue by dt2patel** [hotwax/fulfillment#638]
## What is the motivation for adding/enhancing this feature?

We should remove shipments from the fulfillment screen automatically after rejection even if SOLR has returned the shipment id since it's just a matter of SOLR not being updated.

## What are the acceptance criteria?

- [ ] ...

## Can you complete this feature request by yourself?

- [ ] YES
- [ ] NO

**Comment by dt2patel** [comment]
I think we can close this issue if this will be fixed for good in the new fulfillment APIs. @ravilodhi what do you think?

**Comment by ravilodhi** [comment]
@dt2patel 
Yes it should be fixed in the new fulfillment APIs. We can close the PR and can keep the ticket open to test it against the new Fulfillment app.

**Comment by dt2patel** [comment]
@shubham-namdeo this should be testing along with new fulfillment app backend

**Comment by shubham-namdeo** [comment]
Not working and showing "Failed to update order" even as item is removed after refresh. 
https://jam.dev/c/3071d5c0-f0bb-49ef-90f8-bdcaca21a2b7

With single kit product, order is not rejected at all.
https://jam.dev/c/7b6dc039-eff9-47d6-96be-1c2dd8f53792

**Comment by Varnita-Jain** [comment]
Verified - Working as expected
